lib/helpers.py

```diff
@@ -0,0 +1,33 @@
+import sublime
```

Comment:
This is a generic name.... Why not prefer specific names? Like `sublime_view_helpers`? This will make future us less prone to throw too much shit in the same file.

Reply:
done. although I'm not sure about it being a helper at all... will need another refactoring session